app/views/provider_interface/provider_relationship_permissions_setup/success.html.erb

```diff
@@ -1,17 +1,10 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <div class="govuk-panel govuk-panel--confirmation">
-      <h1 class="govuk-panel__title">
-        Organisational permissions set
-      </h1>
-    </div>
-
-    <p class="govuk-body govuk-!-margin-bottom-4">
-      You can now <%= govuk_link_to 'view applications', provider_interface_applications_path %><% if current_provider_user.authorisation.can_manage_users_for_at_least_one_provider? %> or <%= govuk_link_to 'invite users', provider_interface_provider_users_path %><% end %>.
-    </p>
-
-    <p class="govuk-body govuk-!-margin-bottom-4">
-      You can change organisational permissions in <%= govuk_link_to t('page_titles.provider.account').downcase, provider_interface_account_path %>.
-    </p>
+    <%= govuk_panel(
+      title: 'Organisational permissions set',
+      body: nil,
+    ) %>
+    <p class="govuk-body govuk-!-margin-bottom-4">You can now <%= govuk_link_to 'view applications', provider_interface_applications_path %><% if current_provider_user.authorisation.can_manage_users_for_at_least_one_provider? %> or <%= govuk_link_to 'invite users', provider_interface_provider_users_path %><% end %>.</p>
```

Comment:
This is just a personal preference, but I find putting everything on one line harder to read and work with. Especially in this case with the extra `if` statement.

Reply:
No worries, reverted. I just wish we had a settled set of conventions for markup… but it’s a small, tiny thing.